sw/device/tests/penetrationtests/README.md

```diff
@@ -2,3 +2,28 @@
 
 The penetration testing framework is based on the [Cryptotest](../crypto/cryptotest/README.md) framework.
 The purpose of this framework is to perform side-channel analysis (SCA) and fault injection (FI) attacks on the FPGA as well as on the chip.
+
+To build the penetration testing framework for the ChipWhisperer CW310 FPGA board, run:
+```console
+cd $REPO_TOP
+./bazelisk.sh build //sw/device/tests/penetrationtests/firmware:firmware_fpga_cw310_test_rom
+```
+
+Due to code size memory limitations, the firmware for the chip is split into a
+SCA and a FI binary. To build these binaries, run:
```

Comment:
Question do we want to do 1 line per sentence or wrap at 100 chars in this file? Before this PR it was the former, with this PR it's a bit of both.

I suggest checking the Markdown styleguide re some guidance and then adhere to one style only within this file.

Reply:
Oh sorry about that, it seems that we are not following the markdown style guide in ot-sca at all, hence the confusion on my side.
Fixed.